### ISSUE TITLE
[Feature Request] Version Highlights web page ingame

### ISSUE THREAD

**Issue by xNissX233** [Melledy/Grasscutter#1547]
**Is your feature request related to a problem? Please describe.**
When clicking in the Version Highlights in the game menu, it should redirect you to a html site with the changes of the Genshin version. Since this is unused right now, because it can't get the information from Genshin servers, it would be cool to have a redirection to a local simple webpage that shows the current Grasscutter version changes or the recent commits in the github.

**Describe the solution you'd like**
When clicking this icon 

![Screenshot_20220722_143205](https://user-images.githubusercontent.com/35869848/180441693-c296dd88-e99f-4bda-b868-78d56ab3ecc1.png)

a website with the information of the current state of Grasscutter is displayed. The icon "Cloud Genshin" redirects you to an error webpage.

![Screenshot_20220722_144433](https://user-images.githubusercontent.com/35869848/180441772-1b746998-9639-47af-9c64-daf9b3da3069.png)

So it should be possible to have whatever html you want displayed in this menus. Then, why not a simple website displaying information about Grasscutter? like current bugs, recent commits, latest stable release...

It's just a gimmick tbh, but it should be easy enough.

**Comment by millet6** [comment]
![20220723125407](https://user-images.githubusercontent.com/109255084/180591023-88c9504a-43ab-413e-94fc-4115540ca58b.png)
这是你想要的 实现的效果 （This is what you want to achieve）https://ci.grasscutter.ga/job/Grasscutter-Dev/

**Comment by millet6** [comment]
![20220723130203](https://user-images.githubusercontent.com/109255084/180591158-499f243b-7c2f-4846-8e1b-403552bc04dd.png)
代码正确写法  www.baidu.com改ci.grasscutter.ga/job/Grasscutter-Dev/  （Write the code correctly www.baidu.com to ci.grasscutter.ga/job/grasscutter-dev/）

**Comment by xNissX233** [comment]
Yes, somehow. But that redirects all the errors to that site. I want to redirect only the "Version Highlights" section.

**Comment by millet6** [comment]
https://ci.grasscutter.ga/job/Grasscutter-Dev/changes 你重写网页架构 去侧边栏 （You rewrite the page structure to the sidebar）

**Comment by xNissX233** [comment]
I know that this https://ci.grasscutter.ga/job/Grasscutter-Dev/changes shows exactly what I want. But I want to show this only when the "Version Highlights" icon is clicked, not when UnhandledRequestRouter is called. But I don't know what method do "Version Highlights" calls, since it doesn't show an error site. The error site example is what I want this button to show, but if you click "Version Highlights" you will see a blank page, not the error page, so this button calls something that is not UnhandledRequestRouter I think.

**Comment by xNissX233** [comment]
I made a fork to apply the changes you said, and indeed, it shows the site (broken, but could be fixed) where the error page was before. But still, in the "Version Highlights" section, I only see this:
![Screenshot_20220723_105143](https://user-images.githubusercontent.com/35869848/180598010-f83eee89-bd1c-4e25-80e1-1988606bf92b.png)

**Comment by millet6** [comment]
Version Highlights  能发图吗？这个按扭 Can you send pictures? This button

**Comment by xNissX233** [comment]
> Version Highlights 能发图吗？这个按扭 Can you send pictures? This button

Oh, there is a picture in my first comment, in the section "Describe the solution you'd like", after "When clicking this icon:"

**Comment by millet6** [comment]
Oh, I see.
